src/components/about/sections/ManagementBoardSection/ManagementBoardDescription.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from '@mui/material'
+import { Description } from 'components/about/AboutPage.styled'
+import { useTranslation } from 'next-i18next'
+import { useState } from 'react'
+
+export function ManagementBoardDescritpion({ description }) {
+  const { t } = useTranslation()
+
+  const [show, setShow] = useState(false)
+
+  return (
+    <>
+      <Description variant="body2" height={show ? 'auto' : '200px'}>
+        {description}
+      </Description>
+      <div margin="0 auto" width="100%">
+        <Button variant="outlined" onClick={() => setShow(!show)}>
+          {show ? t('campaigns:cta.see-less') : t('campaigns:cta.see-more')}
+        </Button>
+      </div>
```

Comment:
You probably shouldn't be using a `<div>` and styling it directly. Also I don't exactly get why you need the `margin: 0 auto;` when you are have a `width: 100%`.

If the whole idea of that was to make the button full width there is a property on the MUI Button called `fullWidth` that you can toggle https://mui.com/material-ui/api/button/

Reply:
Keep in mind that `margin` is not really a standard HTML attribute, and here it's used as CSS style.

Probably the whole div wrapper should not be there.